tests/ElmFormat/TestWorld.hs

```diff
@@ -43,20 +44,28 @@ fullStderr state =
         |> reverse
         |> concat
 
-
+{-| Files paths are normalized to allow:
+   - "./elm-package.json" and "package.json" for example to point to the same file
+        (this is required for Elm version autodetection to work correctly in tests)
+   - POSIX paths like "src/test.elm" to also work on Windows in tests
+-}
 instance World (State.State TestWorldState) where
     doesFileExist path =
         do
             state <- State.get
-            return $ Dict.member path (filesystem state)
+            return $ Dict.member (FilePath.normalise path) (filesystem state)
 
     doesDirectoryExist _path =
         return False
 
+    makeAbsolute path =
+        -- wrong but enough for tests
```

Comment:
Is it ok to you? I'm not sure that writing a lot of code for this now is worth it.